hybrasyl/Objects/VisibleObject.cs

```diff
@@ -50,19 +50,21 @@ public class VisibleObject : WorldObject
         public string DisplayText { get; set; }
 
         public string DeathPileOwner { get; set; }
-        public List<string> DeathPileAllowedLooters { get; set; }
-        public DateTime? DeathPileTime { get; set; }
+        public List<string> ItemDropAllowedLooters { get; set; }
+        public DateTime? ItemDropTime { get; set; }
+        public ItemDropType ItemDropType { get; set; }
```

Comment:
These can stay here for now but a future refactor is to clean up this and move it to `Item` instead as it is nonsensical for, say, a `Creature` to have a drop type, or to have allowed looters (since these properties are in `VisibleObject`)

Reply:
Okay. I'll add that to my list.

Would that also apply to DeathPileOwner?

Reply:
Yes.